src/components/LinearProgressWithLabel/LinearProgressWithLabel.tsx

```diff
@@ -0,0 +1,152 @@
+import { Typography, Tooltip, LinearProgress, LinearProgressProps, linearProgressClasses } from '@common/components'
+import { styled } from '@common/theme'
+import { useMemo } from 'react'
+import { StyledComponent } from '@emotion/styled'
+
+/*
```

Comment:
Damit es richtig erkannt wird.
```suggestion
/**
```